startup.cmd

```diff
@@ -50,7 +50,7 @@ SET PYTHONPATH=%TABPY_ROOT%\tabpy-server;%TABPY_ROOT%\tabpy-tools;%PYTHONPATH%
 SET STARTUP_CMD=python tabpy-server\tabpy_server\tabpy.py
 IF [%1] NEQ [] (
     ECHO     Using config file at %TABPY_ROOT%\tabpy-server\tabpy_server\%1
-    SET STARTUP_CMD=%STARTUP_CMD% --config=%1
+    SET STARTUP_CMD=%STARTUP_CMD% --config=%TABPY_ROOT%\tabpy-server\tabpy_server\%1
```

Comment:
I am not sure this is what we want. What happens if I run `startup.cmd c:\Users\ogolovatyi\Desktop\myconfig.conf`

Reply:
I thought the config was restricted to living in tabpy-server/tabpy_server? If not then yeah I don't think this will work